packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -248,7 +248,8 @@ class TextFormField extends FormField<String> {
   /// If null, this widget will create its own [TextEditingController] and
   /// initialize its [TextEditingController.text] with [initialValue].
   final TextEditingController? controller;
-
+  /// {@macro flutter.material.TextField.onChanged}
```

Comment:
This macro doesn't exist, did you mean `flutter.material.editableText.onChanged`?  Or you could create this macro if you want the entire TextField.onChanged docs.